src/app.js

```diff
@@ -207,43 +207,34 @@ class App {
   }
 
   // Start the synchronization
-  startSync (mode, callback) {
+  startSync (mode) {
     this.config.saveMode(mode)
     log.info('Run first synchronisation...')
-    this.sync.start(mode, err => {
-      this.sync.stop(() => {})
-      if (err) {
-        log.error(err)
-      }
-      callback(err)
-    })
+    this.sync.start(mode)
   }
```

Comment:
I may be misunderstanding something, but isn't this supposed to return a promise?